spec/js-api/value/calculation.d.ts.md

```diff
@@ -237,41 +237,48 @@ hashCode(): number;
 
 ### `CalculationInterpolation`
 
-The JS API representation of a Sass [`CalculationInterpolation`].
+A deprecated alternative JS API representation of an unquoted Sass string that's
+always surrounded by parentheses. It's never returned by the Sass compiler, but
+for backwards-compatibility users may still construct it and pass it to the Sass
+compiler.
 
-[`CalculationInterpolation`]: ../../types/calculation.md#types
+> `CalculationInterpolation`s are no longer generated by the Sass compiler,
+> because it can now tell at evaluation time whether an interpolation was
+> originally surrounded by parentheses. However, until we make a breaking
+> revision of the JS API, users may continue to pass `CalculationInterpolation`s
 
 ```ts
 export class CalculationInterpolation implements ValueObject {
 ```
 
 #### `internal`
 
-A private property like [`Value.internal`] that refers to a Sass
-[`CalculationInterpolation`].
+A private property like [`Value.internal`] that refers to a Sass string.
 
 #### Constructor
 
-Creates a Sass [`CalculationInterpolation`] by setting the `value` field to the
-`value` argument and returns it.
+Creates a `CalculationInterpolation` with its `internal` set to an unquoted Sass
```

Comment:
Should this line read without `its`? or maybe it's missing a word after `internal`?